doc/contributing/reviewing-contributions.xml

```diff
@@ -493,6 +493,120 @@
 </screen>
   </example>
  </section>
+ <section xml:id="reviewing-contributions-maintainers">
+  <title>Individual Maintainer List</title>
+
+  <para>
+   When adding users to <filename>maintainers/maintainer-list.nix</filename>, the following checks should be performed:
+  </para>
+
+  <itemizedlist>
+   <listitem>
```

Comment:
It might be useful to add ids to individual list items so we can link them in reviews.

Reply:
Nice idea, I added some!